packages/charts/react-charting/src/components/GroupedVerticalBarChartV2/GroupedVerticalBarChartV2.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { IGroupedVerticalBarChartV2Props } from './GroupedVerticalBarChartV2.types';
+import { GroupedVerticalBarChart } from '../GroupedVerticalBarChart/index';
+import { IGroupedVerticalBarChartData, ILineChartPoints } from '../../types/IDataPoint';
+
+export const GroupedVerticalBarChartV2: React.FunctionComponent<IGroupedVerticalBarChartV2Props> = ({
```

Comment:
instead of creating a new V2 wrapper, could you try creating a new V2 ChartProps and convert v2 props to v1 props internally if V2 properties are present

Reply:
done